README.md

```diff
@@ -1,29 +1,48 @@
 # cdist
 
-**cdist** is a usable configuration management system.
+This is [community maintained](https://github.com/cdist-community)
+fork of Ungleich's [cdist](https://github.com/ungleich/cdist)
+(after [f061fb168ddacc894cb6e9882ff5c8ba002fadd8](https://github.com/ungleich/cdist/commit/f061fb168ddacc894cb6e9882ff5c8ba002fadd8)).
```

Comment:
Please shorten the commit ID to f061fb1.
It is too obtrusive in its full length.